admin/load.php

```diff
@@ -73,7 +73,7 @@ function perflab_render_settings_page() {
  *
  * @param string $hook_suffix The current admin page.
  */
-function perflab_admin_pointer( $hook_suffix ) {
+function perflab_admin_pointer( string $hook_suffix ) {
```

Comment:
Using PHP7 type hinting is a bit of a personal preference. I am personally in favor, but we haven't been doing it for the most part, so I feel this shouldn't be dictated by a tool.

I am not strictly opposed to the change, but it feels out of scope here and IMO should rather be the result of a conscious decision that the team is aware of.

Reply:
ok, I reverted in 2269c6017335c995800d28c0a8948e5c921e8559